### PR TITLE
Turn off summernote airmode for NarrowEditor

### PR DESCRIPTION
**Pull request by trangmei**
Fix https://github.com/Coursemology/coursemology2/issues/3197

### CODE REVIEW

**Comment by trangmei** [comment]
add_comment

Other issues:
1. One extra click is needed to open new comment field. Should I make this field always visible on the NarrowEditor and WideEditor?
2. Summernote toolbar is too big, especially in this view. Should I change styles for Material Summernote component?

**Comment by jeremyyap** [comment]
I think the narrow editor comments are rendered using this library https://github.com/react-bootstrap/react-overlays

Seems that there may be an option to disable close on click outside: https://github.com/react-bootstrap/react-overlays/blob/59978a15b27c7c3d260036a146256176d7a2ff09/src/Overlay.js#L118

Code is here: https://github.com/coursemology-collab/coursemology2/blob/3ec84548b4e78be085603ff5b19e89bd55a03d6f/client/app/bundles/course/assessment/submission/components/ReadOnlyEditor/NarrowEditor.jsx#L75

**Comment by trangmei** [comment]
Clicking on overlay to dismiss would work for simple overlay. But this overlay contains elements that users will interact with.
Removing rootClose might resurrect this old bug: https://github.com/Coursemology/coursemology2/issues/2796#issuecomment-368811715

If you're keen on having popover summernote toolbar on top of Overlay, would using Modal instead of Overlay help?

**Comment by trangmei** [comment]
add_comment_button
ADD COMMENT button added

**Comment by trangmei** [comment]
@jeremyyap if this approach looks ok, i'll write tests for CommentCard:
https://codecov.io/gh/Coursemology/coursemology2/compare/81b756e497d613c6e2b5d5cc01a9235f47f22d3c...a09edf8c0225c6b9631f98972341ff7ad6a3abde/src/client/app/bundles/course/assessment/submission/components/CommentCard.jsx

File: client/app/bundles/course/assessment/submission/components/CommentCard.jsx [hunk 1]
```diff
@@ -105,13 +106,13 @@ export default class CommentCard extends Component {
...
     const { editMode } = this.state;
-    const { editValue, post: { formattedText, id } } = this.props;
+    const { editValue, disableAirMode, post: { formattedText, id } } = this.props;
 
     if (editMode) {
       return (
         <>
           <MaterialSummernote
-            airMode
+            airMode={disableAirMode !== true}
```
Comment (jeremyyap) [inline client/app/bundles/course/assessment/submission/components/CommentCard.jsx]: Can write `airMode={!disableAirMode}`

But this double negative can be confusing to read. I would suggest to invert `disableAirMode` everywhere and use `airMode` prop instead, and set it as `true` using defaultProps.

**Pull request merged**